### PR TITLE
removed the usage of !important and added a stylelint that prohibits the usage of !important

### LINKED ISSUE #184: Remove css !important
We've made some awesome progress so far! Let's go back and clean up a few of our coding shortcuts. 

### Acceptance Criteria

- [ ] Remove all usages of the css rule `!important`.
- [ ] Add a linter rule for preventing future `!important`s

### PR DESCRIPTION
**Pull request by shoebsd31**
## Resolves #184 

I have removed the usage of !important throughout the project. I have also added stylelint to prevent the usage of !important in code.

### How to test

use npm install and then run 
npx stylelint "**/*.{scss,vue}" to run the linting
the rules can be added/modified in the .stylelintrc.json file

### Screenshots

_Screenshots of the changes working_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,9 @@
     "eslint-plugin-vue": "^7.11.1",
```
Comment (Ermish) [inline package.json:26]: please remove the package-lock.json as we are using `yarn` instead of `npm`

**Review by Ermish — CHANGES_REQUESTED** [review]
Awesome! Thanks for the help! Just need to remove the `package-lock.json` file and we're good!

**Comment by ann-kilzer** [comment]
Yes, we're using yarn for this project, so please adjust the instructions.

Otherwise really glad to see this tool addition

File: layouts/default.vue [hunk 2]
```diff
@@ -24,6 +24,7 @@ export default {
   display: flex;
   justify-content: center;
 }
+
```
Comment (ann-kilzer) [inline layouts/default.vue:27]: Please avoid adding unnecessary whitespace changes

File: pages/about.vue [hunk 3]
```diff
@@ -47,9 +47,11 @@
   flex-wrap: wrap;
   justify-content: space-evenly;
 }
+
```
Comment (ann-kilzer) [inline pages/about.vue:50]: same here

File: assets/variables.scss [hunk 4]
```diff
@@ -4,5 +4,5 @@
 // $font-size-root: 20px;
 
 .black-text {
-  color: black !important;
```
Comment (ann-kilzer) [inline assets/variables.scss:7]: 👍🏻

**Comment by shoebsd31** [comment]
> Yes, we're using yarn for this project, so please adjust the instructions.
> 
> Otherwise really glad to see this tool addition

Hi Ann, 

I am sorry I didn't understand, which instructions ?

**Comment by ann-kilzer** [comment]
"How to test" on this ticket references npm.

**Comment by ermish** [comment]
@shoebsd31 she's referring to this merge request is referencing `npm` instead of `yarn` in the `how to test` section. 

I also noticed the `yarn.lock` seems to point everything to `npm` instead of `yarnpkg`. You might want to undo your changes to the `yarn-lock` file and rerun `yarn` to add your new stylelint reference

**Comment by shoebsd31** [comment]
> @shoebsd31 she's referring to this merge request is referencing `npm` instead of `yarn` in the `how to test` section.
> 
> I also noticed the `yarn.lock` seems to point everything to `npm` instead of `yarnpkg`.
> You might want to
> 
> 1. merge the latest changes from `main`
> 2. undo your changes to the `yarn-lock` file
> 3. rerun `yarn` to add your new stylelint reference

got it, I will update the code soon :)

**Comment by ann-kilzer** [comment]
Closed in favor of #218

**Pull request closed without merging**